docs-src/clusters/how-to-integrate-elasticsearch-into-a-temporal-cluster.md

```diff
@@ -8,26 +8,26 @@ tags:
   - filtered-lists
   - visibility
 ssdi:
-- Elasticsearch v8 is supported beginning with Temporal Server version 1.18.0
-- Elasticsearch v7.10 is supported beginning with Temporal Server version 1.17.0
-- Elasticsearch v6.8 is supported through Temporal Server version 1.17.x
-- Elasticsearch v6.8 and v7.10 are explicitly supported with AWS Elasticsearch
+  - Elasticsearch v8 is supported beginning with Temporal Server version 1.18.0
+  - Elasticsearch v7.10 is supported beginning with Temporal Server version 1.17.0
+  - Elasticsearch v6.8 is supported through Temporal Server version 1.17.x
+  - Elasticsearch v6.8 and v7.10 are explicitly supported with AWS Elasticsearch
```

Comment:
```suggestion
  - Elasticsearch v8 is supported beginning with Temporal Server version 1.18.0.
  - Elasticsearch v7.10 is supported beginning with Temporal Server version 1.17.0.
  - Elasticsearch v6.8 is supported through Temporal Server version 1.17._x_.
  - Elasticsearch v6.8 and v7.10 are explicitly supported with AWS Elasticsearch.
```
- Terminal periods added to match periods in other SSDI instances.
- The _x_ in "1.17._x_" styled italic per https://developers.google.com/style/text-formatting#italic.